packages/core/types/@chainlink/index.d.ts

```diff
@@ -147,6 +147,16 @@ declare module '@chainlink/types' {
 
   /* INPUT TYPE VALIDATIONS */
   export type Override = Map<string, Map<string, string>>
+
+  // Includes is an alternative symbol mapping that can be used to represent
+  // the original request, such as wrapped tokens on DEXes.
+  export type Includes = {
+    from: string, // From symbol
+    to: string, // To symbol
+    adapters?: string[], // Array of adapters this applies to
+    inverse?: boolean // If the inverse should be calculated instead
+    tokens?: boolean // If the token addresses should be used instead
+  }
```

Comment:
Can we add this to the README above where `overrideIncludes` is documented? Would make it easier to understand what data format we're dealing with without going into the code.